src/node/desktop/src/assets/locales/fr.json

```diff
@@ -1,113 +1,113 @@
 {
   "common": {
-    "buttonOk": "FR [OK]",
-    "buttonCancel": "FR [Cancel]",
-    "buttonBrowse": "FR [Browse...]",
-    "unknownErrorOccurred": "FR [An unknown error occurred.]"
+    "buttonOk": "OK",
+    "buttonCancel": "Annuler",
+    "buttonBrowse": "Parcourir...",
+    "unknownErrorOccurred": "Une erreur inconnue s'est produite."
   },
   "contextMenu": {
-    "saveImageAsDots": "FR [Save image as...]",
-    "saveImageAs": "FR [Save image as]",
-    "copyImage": "FR [Copy Image]",
-    "reload": "FR [Reload]",
-    "copyImageAddress": "FR [Copy Image Address]",
-    "cut": "FR [Cut]",
-    "copy": "FR [Copy]",
-    "paste": "FR [Paste]",
-    "selectAll": "FR [Select All]",
-    "inspectElement": "FR [Inspect Element]",
-    "errorDownloadingImage": "FR [Error downloading image]",
-    "downloadCancelledMessage": "FR [The download was cancelled.]",
-    "downloadInterruptedMessage": "FR [The download was interrupted.]",
-    "save": "FR [Save]"
+    "saveImageAsDots": "Enregistrer l'image sous...",
+    "saveImageAs": "Enregistrer l'image sous",
+    "copyImage": "Copier l'image",
+    "reload": "Recharger",
+    "copyImageAddress": "Adresse de l'image à copier",
```

Comment:
Does the English mean "address of the image to copy" ? in which case this would be: "Copier l'adresse de l'image"